__tests__/app.test.js

```diff
@@ -161,3 +161,58 @@ describe("GET /api/reviews/:review_id/comments", () => {
     });
   });
 });
```

Comment:
These test are good but there a couple of other things to consider:

- Does the endpoint ignore unnecessary properties on the request?
- What if the review ID is of an invalid data type eg "not-an-id"?

Reply:
Added another test to cover invalid data types i.e when entering a string not a number